alert-policies/browser/AjaxThroughput.yml

```diff
@@ -0,0 +1,19 @@
+name: Ajax Throughput
+description: |+
+  This baseline alert condition evaluates Ajax throughput deviations.
+type: BASELINE
+nrql:
+  query: "SELECT rate(count(*), 1 minute) FROM AjaxRequest WHERE appName like '%' FACET appName"
+baselineDirection: UPPER_AND_LOWER
+terms:
+  - priority: CRITICAL
+    operator: ABOVE
+    threshold: 3
+    thresholdDuration: 500
+    thresholdOccurrences: ALL
+signal:
```

Comment:
same here

Reply:
I removed the `fillOption` and `fillValue` fields and the defaults (`STATIC` and `0`) are OK with me. 👍  
Marking the rest of the comments as resolved as this is the same fields noted across the review!